.github/workflows/ros_ci.yml

```diff
@@ -0,0 +1,51 @@
+name: ROS 1 CI
+
+on:
+  pull_request:
+    branches:
+      - 'develop'
+      - 'master'
```

Comment:
master branch isn't maintained anymore and should be omitted.

Reply:
```suggestion
```